src/app/features/work-view/split/split.component.ts

```diff
@@ -53,6 +54,14 @@ export class SplitComponent implements AfterViewInit {
     this._updatePos(this.pos, false);
     this._renderer.addClass(this.splitTopEl, ANIMATABLE_CLASS);
     this._renderer.addClass(this.splitBottomEl, ANIMATABLE_CLASS);
+
+    this.route.queryParams.subscribe((params) => {
```

Comment:
The split component is more of a dumb ui component, so I think it shouldn't be made aware of the routing and isInbBacklog. Makes more sense to me if the parent component does that and sets `splitPos` accordingly.

Reply:
Sounds good to me, will take care of it.